src/styles/stars.scss

```diff
@@ -0,0 +1,24 @@
+@use './variables' as *;
+
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url(../images/star.svg);
+    background-position: center;
+    background-repeat: no-repeat;
+    height: $star-size;
+    width: $star-size;
+    margin-right: $star-margin;
+
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+
+  @for $i from 1 through 5 {
+    &--#{$i} :nth-child(-n + #{$i}) {
```

Comment:
The selector `&--#{$i} :nth-child(-n + #{$i})` seems incorrect. There should not be a space between the `&` and the colon `:`. It should be `&--#{$i}:nth-child(-n + #{$i})` to ensure the SCSS compiles to the proper CSS class.